app/src/app/ui/overrides/ay11.scss

```diff
@@ -0,0 +1,203 @@
+@mixin ay11-theme() {
+  .adf-pagination {
+    &__perpage-block,
+    &__actualinfo-block {
+
+      .mat-icon-button {
+        left: 3px;
+        &:focus-visible {
+          outline: none;
+        }
+
+        &.cdk-keyboard-focused {
+          .mat-icon {
+            @include rounded-outline();
+          }
+        }
+      }
+    }
+  }
+
+  .mat-icon-button, .mat-button, .mat-checkbox, .mat-raised-button {
+    &:focus-visible {
+      @include rounded-outline();
+    }
+    .mat-ripple-element {
+      display: none;
+    }
+    &.cdk-keyboard-focused {
+      .mat-button-focus-overlay {
+        display: none;
+      }
+      .mat-checkbox-ripple {
+        @include rounded-outline();
+        height: 22px;
+        width: 22px;
+        left: calc(50% - 11px);
+        top: calc(50% - 11px);
+      }
+    }
+  }
+
+  .adf-content-node-selector-content-list .adf-datatable-list .adf-datatable-selected > svg {
+    fill: var(--theme-blue-button-color) !important;
+  }
+
+  .mat-list-base .mat-list-option {
+    &:focus-visible {
+      background-color: white;
+      .mat-list-item-content {
+        @include rounded-border();
+      }
+    }
+    .mat-list-item-content {
+      @include rounded-border(transparent);
+    }
+  }
+
+  .acs-close-members-container {
+    .mat-icon-button {
+      outline: none;
+      line-height: 36px !important;
+      @include rounded-border(transparent);
+      &:focus-visible {
+        @include rounded-border();
+      }
+    }
+  }
+  .adf-search-user-button, .app-search-button {
+    line-height: 36px !important;
+    @include rounded-border(transparent);
+    &:focus-visible {
+      outline: none;
+      @include rounded-border();
+    }
+  }
+
+  .mat-menu-item {
+    .mat-menu-ripple {
+      display: none;
+    }
+    @include rounded-border(transparent);
+    &:focus-visible {
+      @include rounded-border();
+      background-color: white;
+    }
+    &.cdk-keyboard-focused:not([disabled]) {
+      background-color: white;
+    }
+  }
+
+  .adf-toolbar-title {
+    padding: 2px;
+  }
+
+  .mat-option {
+    &.cdk-keyboard-focused {
+      .mat-option-ripple {
+        @include rounded-outline();
+      }
+    }
+  }
+
+  .mat-slide-toggle {
+    .mat-slide-toggle-thumb {
+        width: 15px;
+        height: 15px;
+        transform: translate(20%, 50%);
+        background-color: #484a4d;
+    }
+
+    .mat-slide-toggle-bar {
+      border-radius: 15px;
+      height: 20px;
+    }
+
+    .mat-slide-toggle-thumb-container {
+      top: -5px;
+    }
+
+    &.mat-primary.mat-checked:not(.mat-disabled) {
```

Comment:
I suppose, here empty line is not needed

Reply:
Yeah I've moved it a bit to look nicer